### PR TITLE
chore: Added Trouble Shooting Guide to documentation site

### PR DESCRIPTION
**Pull request by rllyy97**
## Main changes
- Added TSG to docs site
- Adjusted old storybook deploy action to use new `Documentation` label
  - Just adjusted for naming accuracy, was already set up to still work on the new docs it looks like

You can view the new doc here: https://purple-pebble-0e11de71e-2623.westus2.1.azurestaticapps.net/tsg

### CODE REVIEW

**Review by ccastrotrejo — APPROVED** [review]

**Review by Eric-B-Wu — APPROVED** [review]

File: apps/logic-apps-ux-docs/docs/tsg.mdx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+If you're experiencing any issues with Logic Apps Designer, please follow the steps below to help us investigate the issue as well as mitigate the issue for you.
+
+## General mitigation steps
+
+### Fallback to legacy designer
+
+Our current main method of mitigation is to fall back to the now legacy version of Designer. This can be done by using the following URL to navigate to your Logic App. This will hopefully allow you to continue working while we investigate your issue.
+
+**[Link to access legacy designer](https://portal.azure.com/?microsoft_azure_ema_forceFallbackToOldDesigner=true#view)**
```
Comment (preetriti1) [inline apps/logic-apps-ux-docs/docs/tsg.mdx:17]: Can we use the aka.ms/lafallback link?

File: apps/logic-apps-ux-docs/docs/tsg.mdx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+1. Open the developer tools in your browser with the keyboard shortcut `Ctrl+Shift+I`
+2. Select the **Network** tab
+3. Refresh the page and reproduce the issue
+4. Once you have reproduced the issue, right-click anywhere in the grid of network requests, select **Save as HAR with Content**, and save the file to your computer
+5. Your HAR file is now ready to be sent
+
+:::caution Warning on sensitive information
+
+These network logs may contain sensitive information, so please do not share them within GitHub issues. If an engineer requests them, please send them via email. (Should be supplied when requesting the HAR)
```
Comment (preetriti1) [inline apps/logic-apps-ux-docs/docs/tsg.mdx:70]: We should also mention about opening tickets with CSS or incident if this is for CSS and sharing them there, depending on the severity of the issue

File: libs/designer/src/lib/core/actions/bjsworkflow/initialize.ts [hunk 3]
```diff
@@ -265,6 +265,7 @@ export const updateOutputsAndTokens = async (
   let tokens: OutputToken[];
   if (supportsManifest) {
     const manifest = await getOperationManifest(operationInfo);
+    console.log('### manifest', manifest);
```
Comment (preetriti1) [inline libs/designer/src/lib/core/actions/bjsworkflow/initialize.ts:268]: remove

**Comment by preetriti1** [comment]
We should also mention about getting their sessionId along with our extension version

**Review by preetriti1 — APPROVED** [review]

**Pull request merged**